src/tools/genpolicy/policy_samples.json

```diff
@@ -35,6 +35,7 @@
                 "kubernetes/incomplete-init/cockroachdb-statefulset.yaml",
                 "pod/pod-exec.yaml",
                 "pod/pod-lifecycle.yaml",
+                "pod/pod-many-layers.yaml",
```

Comment:
@danmihai1 Could you add this test case to "needs_containerd_pull" category down below? Looks like this sample doesn't work with default pull